superset/connectors/sqla/models.py

```diff
@@ -2042,6 +2042,10 @@ def after_delete(  # pylint: disable=unused-argument
         dataset = (
             session.query(NewDataset).filter_by(sqlatable_id=target.id).one_or_none()
         )
+        for tbl in dataset.tables:
+            if len(tbl.datasets) == 1 and tbl.datasets[0] == dataset:
+                session.delete(tbl)
```

Comment:
Is this needed in this PR or just leftover?

Reply:
This PR is not needed, I just want to pass the CI.